src/SearchableEntity.php

```diff
@@ -41,7 +42,11 @@ public function getSearchableArray()
             $context['groups'] = [Searchable::NORMALIZATION_GROUP];
         }
 
-        return $this->normalizer->normalize($this->entity, Searchable::NORMALIZATION_FORMAT, $context);
+        if ($this->normalizer instanceof NormalizerInterface) {
+            return $this->normalizer->normalize($this->entity, Searchable::NORMALIZATION_FORMAT, $context);
+        } elseif ($this->normalizer instanceof ArrayTransformerInterface) {
+            return $this->normalizer->toArray($this->entity);
```

Comment:
I never really used the JMSSerializerBundle, I used `ArrayTransformerInterface::toArray` because I think it's the equivalent to the `NormalizerInterface::normalize` method but you can't pass a format. Is it easy enough to customize?

Reply:
May `->serialize($objects, 'array')`